llvm/test/CodeGen/AArch64/machine-cp-undef.mir

```diff
@@ -0,0 +1,145 @@
+# RUN: llc -o - %s -O3 -mcpu=apple-m1 -mtriple=arm64-apple-macos | FileCheck %s
```

Comment:
Can this be reduced to just a `-run-pass=machine-cp` test? Please also update update_mir_checks.py.

Reply:
> Can this be reduced to just a `-run-pass=machine-cp` test? Please also update update_mir_checks.py.

I think it's possible, but I'm still trying. After the reduction is complete, I'll switch to update_mir_checks.py.

Reply:
Interestingly, I can use it in with `--start-before`.
First run with `--start-before=machine-cp,0 --stop-before=machine-cp,2`, then run with `--start-before=machine-cp,2 --stop-after=machine-cp,2`. The output is:

```
MCP: Copy is a deletion candidate:   $x8 = ORRXrs $xzr, undef $x0, 0, implicit $w0
MCP: copy is a NOP, removing:   dead renamable $w8 = ORRWrs $wzr, $w0, 0, implicit-def dead $x8
```

Maybe an analysis pass is missing?

Reply:
Resolved. I need add `-mcp-use-is-copy-instr`.